re-add the password host route

wimo7083 opened the pull request:
# Includes the route to the backend without the api
<!-- What does this PR change and why -->

Will still need to add the environment variable for this in k8s.

wimo7083 commented:
I replied in the backend, oops:
https://github.com/OperationCode/operationcode_backend/issues/379#issuecomment-434856132

drewish commented on src/scenes/home/resetPassword/setPassword/setPassword.js:
```diff
@@ -31,19 +31,22 @@ class RequestToken extends Component {
   validatePasswordConfirm = value => value === '' || value === this.state.password;
 
-  handleOnClick = (e) => {
+  handleOnClick = e => {
```
Travis isn't liking this bit
```
./src/scenes/home/resetPassword/setPassword/setPassword.js
  Line 34:  Expected parentheses around arrow function argument having a body with curly braces  arrow-parens
Search for the keywords to learn more about each error.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [build_site] Error 1
```

wimo7083 commented:
Fixed the things, added `api/v1` and hooked into sentry logging so we know sooner when this happens. 

Also added a password change route at `api/v1/update`  if you want to make a form or something for that. You can probably merge this while we wait to push to backend, or you can wait.

kylemh commented on src/scenes/home/resetPassword/requestToken/requestToken.js:
```diff
@@ -27,13 +28,19 @@ class RequestToken extends Component {
         })
-        .catch(() => {
+        .catch((err) => {
+          Raven.setUserContext({
```
this shouldn't actually be necessary, as raven catches all exceptions, but i'll leave it cuz maybe context is helpful

wimo7083 replied:
Context was the only reason I put this in here.

kylemh commented on src/scenes/home/resetPassword/setPassword/setPassword.js:
```diff
@@ -34,19 +34,19 @@ class RequestToken extends Component {
+          this.setState({ success: true, error: null });
+        })
+        .catch(() => {
+          this.setState({ error: 'We were unable to set the password for this email' });
```
Maybe we should add next steps in this message, such as: "please email staff@operationcode.org for help"

wimo7083 replied:
I think the toasts are fairly small, and this is a big amount of text, what would you recommend?

kylemh replied:
Just the next steps then. It will be red indicating a failure. It's not accessible, but in terms of getting a message across, let's just say to email us.

Also, is the toaster fixed height? I would hope that would expand to fit content.
